community/plugins/spring-ai-alibaba-plugin-larksuite-starter/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1 @@
+com.alibaba.cloud.ai.Configure.PluginToolAutoConfiguration
```

Comment:
add new line

Reply:
emm ,I cannot understand what you said.

Reply:
<img width="300" alt="image" src="https://github.com/user-attachments/assets/c29c5a6a-629e-4abd-a52c-84aa5a43b3ca">